core/src/com/unciv/ui/multiplayer/EditFriendScreen.kt

```diff
@@ -0,0 +1,85 @@
+package com.unciv.ui.multiplayer
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.badlogic.gdx.scenes.scene2d.ui.TextField
+import com.unciv.logic.IdChecker
+import com.unciv.logic.multiplayer.FriendList
+import com.unciv.models.translations.tr
+import com.unciv.ui.pickerscreens.PickerScreen
+import com.unciv.ui.popup.ToastPopup
+import com.unciv.ui.popup.YesNoPopup
+import com.unciv.ui.utils.*
+import java.util.*
+
+class EditFriendScreen(selectedFriend: FriendList.Friend, backScreen: ViewFriendsListScreen) : PickerScreen() {
+    init {
+        val friendNameTextField = TextField(selectedFriend.name, skin)
+        val pastePlayerIDButton = "Player ID from clipboard".toTextButton()
+        val playerIDTextField = TextField(selectedFriend.playerID, skin)
+        val deleteFriendButton = "Delete".toTextButton()
+        val friendlist = FriendList()
+
+        topTable.add("Friend name".toLabel()).row()
+        friendNameTextField.messageText = "Please input a name for your friend!".tr()
+        topTable.add(friendNameTextField).pad(10f).padBottom(30f).width(stage.width/2).row()
+
+        pastePlayerIDButton.onClick {
+            playerIDTextField.text = Gdx.app.clipboard.contents
+        }
+
+        topTable.add("Player ID".toLabel()).row()
+        val gameIDTable = Table()
+        playerIDTextField.messageText = "Please input a player ID for your friend!".tr()
+        gameIDTable.add(playerIDTextField).pad(10f).width(2*stage.width/3 - pastePlayerIDButton.width)
+        gameIDTable.add(pastePlayerIDButton)
+        topTable.add(gameIDTable).padBottom(30f).row()
+
+        deleteFriendButton.onClick {
+            val askPopup = YesNoPopup("Are you sure you want to this friend?", {
```

Comment:
The word "delete" is missing in the sentence.
`Are you sure you want to delete this friend?"` would be right.

Reply:
I can't imagine how I let this happen after reading every word twice. Maybe I deleted it when I cleaned up :))